nbed/ham_builder.py

```diff
@@ -303,12 +304,24 @@ def _reduce_active_space(
             )
         ]
 
-        if self.scf_method.mo_occ.ndim == 1:
+        # For some reason the UHF mo_occ is stored as a one-dimensional array of dtype=object, 
+        # where each entry is itself an array. Therefore, ndim=1, when it should be 2. Old code:
+        # if self.scf_method.mo_occ.ndim == 1:
```

Comment:
You can cut out the old code too, my crimes are well enough documented by github 🥲

Reply:
resolved in [5bd6395](https://github.com/UCL-CCS/Nbed/pull/130/commits/5bd6395f834a3dc3faea3741083c293d666619e1)